test/src/test/java/hudson/model/ActionableTest.java

```diff
@@ -0,0 +1,54 @@
+package hudson.model;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ActionableTest {
```

Comment:
This is not using `JenkinsRule` so it should rather be in `core/src/test/java/hudson/model/ActionableTest.java`.